src/monattack.cpp

```diff
@@ -3004,14 +3010,24 @@ void mattack::chickenbot(monster *z, int index)
             }
             return;
         }
-        cap = target->power_rating() - 1;
     }
 
+    int cap = target->power_rating() - 1;
+    monster *mon = dynamic_cast< monster* >( target );
+    // Their attitude to us and not ours to them, so that bobcats won't get gunned down
+    // Only monster-types for now - assuming humans are smart enough not to make it obvious
+    // Unless damaged - then everything is hostile
+    if( z->hp == z->type->hp ||
```

Comment:
Unless I've got my C++ totally backwards, this makes the tankbot *more* aggressive when it's at full health.  Gonna make that a <= in the merge.